include/openssl/ui.h

```diff
@@ -266,14 +266,12 @@ DEFINE_STACK_OF(UI_STRING)
  * The different types of strings that are currently supported. This is only
  * needed by method authors.
  */
-enum UI_string_types {
-    UIT_NONE = 0,
-    UIT_PROMPT,                 /* Prompt for a string */
-    UIT_VERIFY,                 /* Prompt for a string and verify */
-    UIT_BOOLEAN,                /* Prompt for a yes/no response */
-    UIT_INFO,                   /* Send info to the user */
-    UIT_ERROR                   /* Send an error message to the user */
-};
```

Comment:
Uhmmm...  We have a UI_METHOD in `apps/apps.c`.  I know that engine-pkcs11 have a variant of the same thing in one of their test programs.  I know Windows folks that have written variants that prompt for a password using a dialog box (that's actually the basis for its design).